src/backend/routers/post.py

```diff
@@ -78,6 +78,24 @@ async def create_post(
     return response
 
 
+@router.get("/search")
+async def search_posts(
+    q: str = Query(),
+) -> Page[Post]:
+    """Gets posts based on specified query.
+
+    Matching is done using Full Text Search
+    """
+    words = q.split()
+    search_phrase = " & ".join(words)
+    query = (
+        'SELECT * from "Post" WHERE to_tsvector("title") @@ to_tsquery($1)'
+        ' ORDER BY "created_at" LIMIT 20'
+    )
+    posts = await Post.prisma().query_raw(query, search_phrase)
+    return Page(data=posts, count=len(posts), cursor_id=None)
```

Comment:
Feels a bit weird. I think we should paginate results? But we'd not be able to use the paginate function directly here because of the raw query I guess.

Reply:
We're limiting the number of posts returned to 20

Reply:
What if there are more than 20 search results?

Reply:
The query just won't return those posts now

Reply:
Yes, that's the point of limiting.